draft-ietf-acme-acme.md

```diff
@@ -160,33 +160,31 @@ from the client.
 # Protocol Overview
 
 ACME allows a client to request certificate management actions using a set of
-JSON messages carried over HTTPS.   In some ways, ACME functions much like a
-traditional CA, in which a user creates an account, adds identifiers to that
-account (proving control of the domains), and requests certificate issuance for
-those domains while logged in to the account.
+JSON messages carried over HTTPS.   In many ways, ACME functions much like a
+traditional CA, in which a user creates an account, requests a certificate,
+and proves control of the domains in that certificate in order for the CA to
+sign the requested certificate.
 
-In ACME, the account is represented by an account key pair.  The "add a domain"
```

Comment:
I like this paragraph, because it clarifies how ACME maps to existing CA operations.  Why are you deleting it?